src/utils.py

```
@@ -13,27 +14,27 @@
     {
         "“": '"',
         "”": '"',
-        "‘": "'",
+        "‘": "'",  # noqa: RUF001
         "’": "'",
```

Ruff (0.14.10)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)